config/settings/prod.py

```
@@ -0,0 +1,36 @@
+
+DB_ENGINE = env_str("DB_ENGINE", "django.db.backends.sqlite3")
+DB_NAME = env_str("DB_NAME", str(BASE_DIR / "prod_sim_db.sqlite3"))
+DB_USER = env_str("DB_USER", "")
+DB_PASSWORD = env_str("DB_PASSWORD", "")
+DB_HOST = env_str("DB_HOST", "")
+DB_PORT = env_str("DB_PORT", "")
+
+DATABASES = {
+    "default": {
+        "ENGINE": DB_ENGINE,
+        "NAME": DB_NAME if DB_ENGINE.endswith("sqlite3") else DB_NAME,
```

Ruff (0.12.2)

Useless `if`-`else` condition

(RUF034)